app/src/main/java/com/example/korail_aos/data/service/KorailService.kt

```diff
@@ -1,12 +1,17 @@
 package com.example.korail_aos.data.service
 
 import com.example.korail_aos.data.entity.request.TicketRequest
+import com.example.korail_aos.remote.ResponseMainDto
+import retrofit2.Call
 import retrofit2.http.Body
+import retrofit2.http.GET
 import retrofit2.http.POST
 
 interface KorailService {
     @POST("api/ticket")
     suspend fun postTicket(
         @Body body: TicketRequest
     )
+    @GET("/api/ticket")
+    fun getTicket(): Call<ResponseMainDto>
```

Comment:
```suggestion
    fun getTicket(): Call<BaseResponse<List<Data>>>
```
4. ... 이런 식으로 하면 코드 중복을 없앨 수 있습니다